app/src/components/FormElement/FormElement.js

```diff
@@ -32,20 +23,15 @@ export const FormElement = ({
           type="text"
           placeholder="Shorten a link here..."
           onChange={handleInput}
-          className={`${isVisible ? true : ""}`}
+          className={`${error ? true : ""}`}
         />
-        {isVisible ? (
+        {error ? (
```

Comment:
Do not want to render empty string if there is no error. 

Did you try to {error && <ErrorParagraph>} ? You will avoid the ternary operator